mysql-test/std_data/dd/sdi/upgrade/mysql.json

```diff
@@ -23361,6 +23362,7 @@
         "schema_ref": "mysql",
         "se_private_id":NNN,
         "engine": "InnoDB",
+        "last_checked_for_upgrade_version_id": X,
```

Comment:
Is it upstream, or if it is not upstream, why we are different?

Reply:
as above